include/reelmagic.h

```diff
@@ -36,17 +36,6 @@ struct ReelMagic_VideoMixerMPEGProvider {
 	virtual const ReelMagic_PlayerAttributes& GetAttrs() const     = 0;
 };
 
-void ReelMagic_RENDER_SetPal(uint8_t entry, uint8_t red, uint8_t green, uint8_t blue);
-
-void ReelMagic_RENDER_SetSize(const uint16_t width, const uint16_t height,
-                              const bool double_width, const bool double_height,
-                              const Fraction& pixel_aspect_ratio,
-                              const uint8_t bits_per_pixel,
-                              const double frames_per_second);
-
-bool ReelMagic_RENDER_StartUpdate(void);
-// void ReelMagic_RENDER_EndUpdate(bool abort);
-// void ReelMagic_RENDER_DrawLine(const void *src);
```

Comment:
Are these unused? If they're internal-only, ideally we should change their names (by convention, static functions are in `lower_snake_case` and they're not prefixed by the module name; e.g. `RENDER_DoStuff` would become `do_stuff`).

Reply:
Are you sure removing these is ok? Just removed them in `main` and I'm getting compilation errors:

```
FAILED: src/hardware/libhardware.a.p/vga_other.cpp.o
ccache c++ -Isrc/hardware/libhardware.a.p -Isrc/hardware -I../../src/hardware -I../../include -I. -I../.. -Isrc/libs/ghc -I../../src/libs/ghc -Isubprojects/iir1-1.9.3 -I../../subprojects/iir1-1.9.3 -Isrc/libs/loguru -I../../src/libs/loguru -I/opt/local/include/slirp -I/opt/local/include/glib-2.0 -I/opt/local/lib/glib-2.0/include -I/opt/local/include -I/opt/local/include/libpng16 -I/opt/local/include/SDL2 -fcolor-diagnostics -Wall -Winvalid-pch -Wextra -Wpedantic -std=c++17 -O0 -g -Wno-unknown-pragmas -fpch-preprocess -Weffc++ -Wextra-semi -Wno-deprecated-declarations -fPIC -DLOGURU_STACKTRACES=1 -DLOGURU_USE_LOCALE=0 -D_THREAD_SAFE -MD -MQ src/hardware/libhardware.a.p/vga_other.cpp.o -MF src/hardware/libhardware.a.p/vga_other.cpp.o.d -o src/hardware/libhardware.a.p/vga_other.cpp.o -c ../../src/hardware/vga_other.cpp
../../src/hardware/vga_other.cpp:585:4: error: use of undeclared identifier 'ReelMagic_RENDER_SetPal'
                        RENDER_SetPal(index, r, g, b);
                        ^
../../src/hardware/reelmagic/vga_passthrough.h:26:30: note: expanded from macro 'RENDER_SetPal'
#define RENDER_SetPal        ReelMagic_RENDER_SetPal
                             ^
1 error generated.
[11/19] Compiling C++ object src/hardware/libhardware.a.p/reelmagic_video_mixer.cpp.o
ninja: build stopped: subcommand failed.
```

Reply:
Hmm, the branch itself compiles fine locally here. Not quite sure what's going on...

Reply:
I think @kcgen did not want to change the code much so it's easier to merge further improvements. But my understanding is that the original author will update the version in our repo (if he'll ever get to it). I vaguely remember some correspondence between @kcgen and the author about that. So probably it's okay to make a few judicious improvements to it (and I had to touch it too a bit because of my recent image capture / video related refactorings & improvements).